fixtures/fixture.py

```diff
@@ -137,6 +137,8 @@ def setup(setup_test_repo, setup_test_products):
         options.add_argument('--headless')
         options.add_argument('--no-sandbox')
         options.add_argument('--window-size=1920,1080')
+        options.add_argument('--no-sandbox')
+        options.add_argument('--disable-dev-shm-usage')
```

Comment:
this option also says that it will slow down the tests as per the article shared, let's give it a try though and --no-sandbox is already there, please remove the repetition.